### PR TITLE
Fix `package` icon

### PR DESCRIPTION
**Pull request by danielbayley**
Centre line is extended too far.

### CODE REVIEW

File: icons/package.svg [hunk 1]
```diff
@@ -9,8 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
-  <path d="M21 16V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l7-4A2 2 0 0 0 21 16z" />
-  <polyline points="3.29 7 12 12 20.71 7" />
-  <line x1="12" y1="22" x2="12" y2="12" />
+  <path d="m16.5 9.4-8.6-5M21 16V8c0-.7-.4-1.4-1-1.7l-7-4c-.6-.4-1.4-.4-2 0l-7 4c-.6.3-1 1-1 1.7v8c0 .7.4 1.4 1 1.7l7 4c.6.4 1.4.4 2 0l7-4c.6-.3 1-1 1-1.7z"/>
+  <path d="m3.3 7 8.7 5 8.7-5M12 22V12"/>
```
Comment (karsa-mistmere) [inline icons/package.svg:13]: So it does, nice catch. :)

The paths should remain separate though:
```suggestion
  <path d="M16.5 9.4 7.55 4.24" />
  <path d="M21 16V8a2 2 0 0 0-1-1.73l-7-4a2 2 0 0 0-2 0l-7 4A2 2 0 0 0 3 8v8a2 2 0 0 0 1 1.73l7 4a2 2 0 0 0 2 0l7-4A2 2 0 0 0 21 16Z" />
  <path d="M3.29 7 12 12l8.71-5" />
  <path d="M12 22V12" />
```

File: icons/package.svg [hunk 2]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.8" y2="4.4" />
```
Comment (karsa-mistmere) [inline icons/package.svg:12]: It now doesn't extend far enough:
image

File: icons/package.svg [hunk 3]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.6" y2="4.2" />
```
Comment (karsa-mistmere) [inline icons/package.svg:12]: Now it does still extend by a tiny bit. 😅 
```suggestion
  <line x1="16.5" y1="9.4" x2="7.55" y2="4.24" />
```

Comment (danielbayley) [reply icons/package.svg:12]: Yeah, that’s why I made it less… 🙈

File: icons/package.svg [hunk 4]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <line x1="16.5" y1="9.4" x2="7.5" y2="4.21" />
+  <line x1="16.5" y1="9.4" x2="7.9" y2="4.4" />
```
Comment (karsa-mistmere) [inline icons/package.svg:12]: Still not OK, I'm afraid.
image

Please just believe me and apply the suggestion already. :D
```suggestion
  <line x1="16.5" y1="9.4" x2="7.55" y2="4.24" />
```

**Comment by karsa-mistmere** [comment]
Thanks, this is good to go!

**Pull request merged**